config/environments/development.rb

```diff
@@ -65,10 +65,11 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = false
 
   # Suppress logger output for asset requests.
   config.assets.quiet = true
+  config.assets.check_precompiled_asset = false
```

Comment:
Sorry I guess I didn't save my comment in the previous review, but you shouldn't change these options here. (There are special cases where you may want to, but even then you should always switch them back). We pretty much always want debug on in development so that JS/CSS is easier to debug.